src/lib/network/network_autoyast.rb

```diff
@@ -250,9 +250,13 @@ def rename_lan_item(item, name_to, attr = nil, key = nil)
 
         # make sure that we base renaming on defined attribute with value given in AY profile
         LanItems.ReplaceItemUdev(current_attr, attr, key)
+      elsif attr.nil? ^ key.nil? # xor
```

Comment:
JFYI `^` is bitwise operator, so if you do not have boolean only ( not this case ) it can produce funny results like:
```
irb(main):002:0> true ^ 1
=> false
```